### DISCUSSION TITLE
Error Granite.AccelLabel

### DISCUSSION THREAD

**Discussion by tmanar** [phase1geo/Minder#306] [Q&A]
Hello,
I am using debian buster;
I have all the dependencies (also valac 0.48.14-);
when, in minder folder, I ./app run or sudo ./app install, I have the error below.
Could you help me resolve the problem?
Thanks

"
user@debian:~/Desktop/Minder-master/Minder-master$ ./app run
Directory already configured.

Just run your build command (e.g. ninja) and Meson will regenerate as necessary.
If ninja fails, run "ninja reconfigure" or "meson --reconfigure"
to force Meson to regenerate.

If build failures persist, run "meson setup --wipe" to rebuild from scratch
using the same options as passed when configuring the build.
To change option values, run "meson configure" instead.
[1/199] Compiling Vala source ../s...ts/assert.vala ../tests/test.vala.
FAILED: com.github.phase1geo.minder@exe/src/animator/AnimatorAction.c com.github.phase1geo.minder@exe/src/animator/AnimatorNodes.c com.github.phase1geo.minder@exe/src/animator/AnimatorPan.c com.github.phase1geo.minder@exe/src/animator/AnimatorPanScale.c com.github.phase1geo.minder@exe/src/animator/AnimatorPositions.c com.github.phase1geo.minder@exe/src/animator/AnimatorScale.c com.github.phase1geo.minder@exe/src/exports/ExportCSV.c com.github.phase1geo.minder@exe/src/exports/ExportFreemind.c com.github.phase1geo.minder@exe/src/exports/ExportFreeplane.c com.github.phase1geo.minder@exe/src/exports/ExportImage.c com.github.phase1geo.minder@exe/src/exports/ExportMarkdown.c com.github.phase1geo.minder@exe/src/exports/ExportMermaid.c com.github.phase1geo.minder@exe/src/exports/ExportOPML.c com.github.phase1geo.minder@exe/src/exports/ExportOrgMode.c com.github.phase1geo.minder@exe/src/exports/ExportOutliner.c com.github.phase1geo.minder@exe/src/exports/ExportPDF.c com.github.phase1geo.minder@exe/src/exports/ExportPlantUML.c com.github.phase1geo.minder@exe/src/exports/ExportPNG.c com.github.phase1geo.minder@exe/src/exports/ExportPortableMinder.c com.github.phase1geo.minder@exe/src/exports/ExportPrint.c com.github.phase1geo.minder@exe/src/exports/ExportSVG.c com.github.phase1geo.minder@exe/src/exports/ExportText.c com.github.phase1geo.minder@exe/src/exports/ExportXMind.c com.github.phase1geo.minder@exe/src/exports/ExportYed.c com.github.phase1geo.minder@exe/src/layouts/LayoutDown.c com.github.phase1geo.minder@exe/src/layouts/LayoutHorizontal.c com.github.phase1geo.minder@exe/src/layouts/LayoutLeft.c com.github.phase1geo.minder@exe/src/layouts/LayoutManual.c com.github.phase1geo.minder@exe/src/layouts/LayoutRight.c com.github.phase1geo.minder@exe/src/layouts/LayoutUp.c com.github.phase1geo.minder@exe/src/layouts/LayoutVertical.c com.github.phase1geo.minder@exe/src/parsers/MarkdownParser.c com.github.phase1geo.minder@exe/src/parsers/TaggerParser.c com.github.phase1geo.minder@exe/src/parsers/UrlParser.c com.github.phase1geo.minder@exe/src/styles/LinkTypeCurved.c com.github.phase1geo.minder@exe/src/styles/LinkTypeSquared.c com.github.phase1geo.minder@exe/src/styles/LinkTypeStraight.c com.github.phase1geo.minder@exe/src/styles/NodeBorderBracket.c com.github.phase1geo.minder@exe/src/styles/NodeBorderNone.c com.github.phase1geo.minder@exe/src/styles/NodeBorderPill.c com.github.phase1geo.minder@exe/src/styles/NodeBorderRounded.c com.github.phase1geo.minder@exe/src/styles/NodeBorderSquared.c com.github.phase1geo.minder@exe/src/styles/NodeBorderUnderlined.c com.github.phase1geo.minder@exe/src/themes/ThemeDark.c com.github.phase1geo.minder@exe/src/themes/ThemeDefault.c com.github.phase1geo.minder@exe/src/themes/ThemeSolarizedDark.c com.github.phase1geo.minder@exe/src/themes/ThemeSolarizedLight.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectedNode.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionAdd.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionChange.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionColor.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionDelete.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionNote.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionStickerAdd.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionStickerChange.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionStickerRemove.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionTitle.c com.github.phase1geo.minder@exe/src/undo_actions/UndoConnectionsDelete.c com.github.phase1geo.minder@exe/src/undo_actions/UndoGroupAdd.c com.github.phase1geo.minder@exe/src/undo_actions/UndoGroupRemove.c com.github.phase1geo.minder@exe/src/undo_actions/UndoGroupsColor.c com.github.phase1geo.minder@exe/src/undo_actions/UndoGroupsMerge.c com.github.phase1geo.minder@exe/src/undo_actions/UndoGroupsRemove.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeAddParent.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeAttach.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeBalance.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeCut.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeDelete.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeDetach.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeFolds.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeImage.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeInsert.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeLayout.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeLink.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeLinkColor.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeMove.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeName.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeNote.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodePaste.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeReparent.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeReparentLinkColor.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeReplace.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeResize.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeSort.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeStickerAdd.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeStickerChange.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeStickerRemove.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodeTask.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesAlign.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesCut.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesDelete.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesGroup.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesInsert.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesLink.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesLinkColor.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesRandLinkColor.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesReparentLinkColor.c com.github.phase1geo.minder@exe/src/undo_actions/UndoNodesReplace.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStickerAdd.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStickerChange.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStickerMove.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStickerRemove.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStickerResize.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleAffectChange.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleBranchMargin.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleChange.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleConnectionArrow.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleConnectionDash.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleConnectionFont.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleConnectionLineWidth.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleConnectionPadding.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleConnectionTitleWidth.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleLinkArrow.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleLinkDash.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleLinkType.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleLinkWidth.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodeBorder.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodeBorderwidth.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodeFill.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodeFont.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodeMargin.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodeMarkup.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodePadding.c com.github.phase1geo.minder@exe/src/undo_actions/UndoStyleNodeWidth.c com.github.phase1geo.minder@exe/src/undo_actions/UndoTextClearTags.c com.github.phase1geo.minder@exe/src/undo_actions/UndoTextDelete.c com.github.phase1geo.minder@exe/src/undo_actions/UndoTextInsert.c com.github.phase1geo.minder@exe/src/undo_actions/UndoTextMultiInsert.c com.github.phase1geo.minder@exe/src/undo_actions/UndoTextReplace.c com.github.phase1geo.minder@exe/src/undo_actions/UndoTextTagAdd.c com.github.phase1geo.minder@exe/src/undo_actions/UndoTextTagRemove.c com.github.phase1geo.minder@exe/src/About.c com.github.phase1geo.minder@exe/src/Animator.c com.github.phase1geo.minder@exe/src/Application.c com.github.phase1geo.minder@exe/src/Bezier.c com.github.phase1geo.minder@exe/src/CanvasText.c com.github.phase1geo.minder@exe/src/Clipboard.c com.github.phase1geo.minder@exe/src/Connection.c com.github.phase1geo.minder@exe/src/ConnectionInspector.c com.github.phase1geo.minder@exe/src/ConnectionMenu.c com.github.phase1geo.minder@exe/src/Connections.c com.github.phase1geo.minder@exe/src/ConnectionsMenu.c com.github.phase1geo.minder@exe/src/CurrentInspector.c com.github.phase1geo.minder@exe/src/Document.c com.github.phase1geo.minder@exe/src/DrawArea.c com.github.phase1geo.minder@exe/src/DynamicNotebook.c com.github.phase1geo.minder@exe/src/EmptyInspector.c com.github.phase1geo.minder@exe/src/EmptyMenu.c com.github.phase1geo.minder@exe/src/Export.c com.github.phase1geo.minder@exe/src/Exporter.c com.github.phase1geo.minder@exe/src/Exports.c com.github.phase1geo.minder@exe/src/FormattedText.c com.github.phase1geo.minder@exe/src/GroupsMenu.c com.github.phase1geo.minder@exe/src/HtmlToMarkdown.c com.github.phase1geo.minder@exe/src/ImageEditor.c com.github.phase1geo.minder@exe/src/ImageManager.c com.github.phase1geo.minder@exe/src/Layout.c com.github.phase1geo.minder@exe/src/Layouts.c com.github.phase1geo.minder@exe/src/LinkDash.c com.github.phase1geo.minder@exe/src/LinkType.c com.github.phase1geo.minder@exe/src/MainWindow.c com.github.phase1geo.minder@exe/src/MapInspector.c com.github.phase1geo.minder@exe/src/Node.c com.github.phase1geo.minder@exe/src/NodeAlign.c com.github.phase1geo.minder@exe/src/NodeBorder.c com.github.phase1geo.minder@exe/src/NodeGroup.c com.github.phase1geo.minder@exe/src/NodeGroups.c com.github.phase1geo.minder@exe/src/NodeImage.c com.github.phase1geo.minder@exe/src/NodeInspector.c com.github.phase1geo.minder@exe/src/NodeMenu.c com.github.phase1geo.minder@exe/src/NodesMenu.c com.github.phase1geo.minder@exe/src/NoteView.c com.github.phase1geo.minder@exe/src/Partitioner.c com.github.phase1geo.minder@exe/src/Preferences.c com.github.phase1geo.minder@exe/src/QuickEntry.c com.github.phase1geo.minder@exe/src/Selection.c com.github.phase1geo.minder@exe/src/Sticker.c com.github.phase1geo.minder@exe/src/Stickers.c com.github.phase1geo.minder@exe/src/StickerInspector.c com.github.phase1geo.minder@exe/src/Style.c com.github.phase1geo.minder@exe/src/StyleInspector.c com.github.phase1geo.minder@exe/src/Styles.c com.github.phase1geo.minder@exe/src/Tagger.c com.github.phase1geo.minder@exe/src/TextCompletion.c com.github.phase1geo.minder@exe/src/TextMenu.c com.github.phase1geo.minder@exe/src/TextParser.c com.github.phase1geo.minder@exe/src/Theme.c com.github.phase1geo.minder@exe/src/ThemeEditor.c com.github.phase1geo.minder@exe/src/Themes.c com.github.phase1geo.minder@exe/src/UndoBuffer.c com.github.phase1geo.minder@exe/src/UndoItem.c com.github.phase1geo.minder@exe/src/UndoTextBuffer.c com.github.phase1geo.minder@exe/src/UndoTextItem.c com.github.phase1geo.minder@exe/src/UrlEditor.c com.github.phase1geo.minder@exe/src/Utils.c com.github.phase1geo.minder@exe/tests/assert.c com.github.phase1geo.minder@exe/tests/test.c 
valac -C --debug --debug --define=GTK322 --define=VALAC048 --vapidir /home/x/Desktop/Minder-master/Minder-master/vapi --disable-warnings --pkg gtksourceview-3.0 --pkg libarchive --pkg libxml-2.0 --pkg gtk+-3.0 --pkg granite --pkg gee-0.8 --pkg cairo --pkg glib-2.0 --pkg gobject-2.0 --color=always --directory com.github.phase1geo.minder@exe --basedir ../ --gresources=../data/com.github.phase1geo.minder.gresource.xml /home/x/Desktop/Minder-master/Minder-master/vapi/libmarkdown.vapi ../src/animator/AnimatorAction.vala ../src/animator/AnimatorNodes.vala ../src/animator/AnimatorPan.vala ../src/animator/AnimatorPanScale.vala ../src/animator/AnimatorPositions.vala ../src/animator/AnimatorScale.vala ../src/exports/ExportCSV.vala ../src/exports/ExportFreemind.vala ../src/exports/ExportFreeplane.vala ../src/exports/ExportImage.vala ../src/exports/ExportMarkdown.vala ../src/exports/ExportMermaid.vala ../src/exports/ExportOPML.vala ../src/exports/ExportOrgMode.vala ../src/exports/ExportOutliner.vala ../src/exports/ExportPDF.vala ../src/exports/ExportPlantUML.vala ../src/exports/ExportPNG.vala ../src/exports/ExportPortableMinder.vala ../src/exports/ExportPrint.vala ../src/exports/ExportSVG.vala ../src/exports/ExportText.vala ../src/exports/ExportXMind.vala ../src/exports/ExportYed.vala ../src/layouts/LayoutDown.vala ../src/layouts/LayoutHorizontal.vala ../src/layouts/LayoutLeft.vala ../src/layouts/LayoutManual.vala ../src/layouts/LayoutRight.vala ../src/layouts/LayoutUp.vala ../src/layouts/LayoutVertical.vala ../src/parsers/MarkdownParser.vala ../src/parsers/TaggerParser.vala ../src/parsers/UrlParser.vala ../src/styles/LinkTypeCurved.vala ../src/styles/LinkTypeSquared.vala ../src/styles/LinkTypeStraight.vala ../src/styles/NodeBorderBracket.vala ../src/styles/NodeBorderNone.vala ../src/styles/NodeBorderPill.vala ../src/styles/NodeBorderRounded.vala ../src/styles/NodeBorderSquared.vala ../src/styles/NodeBorderUnderlined.vala ../src/themes/ThemeDark.vala ../src/themes/ThemeDefault.vala ../src/themes/ThemeSolarizedDark.vala ../src/themes/ThemeSolarizedLight.vala ../src/undo_actions/UndoConnectedNode.vala ../src/undo_actions/UndoConnectionAdd.vala ../src/undo_actions/UndoConnectionChange.vala ../src/undo_actions/UndoConnectionColor.vala ../src/undo_actions/UndoConnectionDelete.vala ../src/undo_actions/UndoConnectionNote.vala ../src/undo_actions/UndoConnectionStickerAdd.vala ../src/undo_actions/UndoConnectionStickerChange.vala ../src/undo_actions/UndoConnectionStickerRemove.vala ../src/undo_actions/UndoConnectionTitle.vala ../src/undo_actions/UndoConnectionsDelete.vala ../src/undo_actions/UndoGroupAdd.vala ../src/undo_actions/UndoGroupRemove.vala ../src/undo_actions/UndoGroupsColor.vala ../src/undo_actions/UndoGroupsMerge.vala ../src/undo_actions/UndoGroupsRemove.vala ../src/undo_actions/UndoNodeAddParent.vala ../src/undo_actions/UndoNodeAttach.vala ../src/undo_actions/UndoNodeBalance.vala ../src/undo_actions/UndoNodeCut.vala ../src/undo_actions/UndoNodeDelete.vala ../src/undo_actions/UndoNodeDetach.vala ../src/undo_actions/UndoNodeFolds.vala ../src/undo_actions/UndoNodeImage.vala ../src/undo_actions/UndoNodeInsert.vala ../src/undo_actions/UndoNodeLayout.vala ../src/undo_actions/UndoNodeLink.vala ../src/undo_actions/UndoNodeLinkColor.vala ../src/undo_actions/UndoNodeMove.vala ../src/undo_actions/UndoNodeName.vala ../src/undo_actions/UndoNodeNote.vala ../src/undo_actions/UndoNodePaste.vala ../src/undo_actions/UndoNodeReparent.vala ../src/undo_actions/UndoNodeReparentLinkColor.vala ../src/undo_actions/UndoNodeReplace.vala ../src/undo_actions/UndoNodeResize.vala ../src/undo_actions/UndoNodeSort.vala ../src/undo_actions/UndoNodeStickerAdd.vala ../src/undo_actions/UndoNodeStickerChange.vala ../src/undo_actions/UndoNodeStickerRemove.vala ../src/undo_actions/UndoNodeTask.vala ../src/undo_actions/UndoNodesAlign.vala ../src/undo_actions/UndoNodesCut.vala ../src/undo_actions/UndoNodesDelete.vala ../src/undo_actions/UndoNodesGroup.vala ../src/undo_actions/UndoNodesInsert.vala ../src/undo_actions/UndoNodesLink.vala ../src/undo_actions/UndoNodesLinkColor.vala ../src/undo_actions/UndoNodesRandLinkColor.vala ../src/undo_actions/UndoNodesReparentLinkColor.vala ../src/undo_actions/UndoNodesReplace.vala ../src/undo_actions/UndoStickerAdd.vala ../src/undo_actions/UndoStickerChange.vala ../src/undo_actions/UndoStickerMove.vala ../src/undo_actions/UndoStickerRemove.vala ../src/undo_actions/UndoStickerResize.vala ../src/undo_actions/UndoStyleAffectChange.vala ../src/undo_actions/UndoStyleBranchMargin.vala ../src/undo_actions/UndoStyleChange.vala ../src/undo_actions/UndoStyleConnectionArrow.vala ../src/undo_actions/UndoStyleConnectionDash.vala ../src/undo_actions/UndoStyleConnectionFont.vala ../src/undo_actions/UndoStyleConnectionLineWidth.vala ../src/undo_actions/UndoStyleConnectionPadding.vala ../src/undo_actions/UndoStyleConnectionTitleWidth.vala ../src/undo_actions/UndoStyleLinkArrow.vala ../src/undo_actions/UndoStyleLinkDash.vala ../src/undo_actions/UndoStyleLinkType.vala ../src/undo_actions/UndoStyleLinkWidth.vala ../src/undo_actions/UndoStyleNodeBorder.vala ../src/undo_actions/UndoStyleNodeBorderwidth.vala ../src/undo_actions/UndoStyleNodeFill.vala ../src/undo_actions/UndoStyleNodeFont.vala ../src/undo_actions/UndoStyleNodeMargin.vala ../src/undo_actions/UndoStyleNodeMarkup.vala ../src/undo_actions/UndoStyleNodePadding.vala ../src/undo_actions/UndoStyleNodeWidth.vala ../src/undo_actions/UndoTextClearTags.vala ../src/undo_actions/UndoTextDelete.vala ../src/undo_actions/UndoTextInsert.vala ../src/undo_actions/UndoTextMultiInsert.vala ../src/undo_actions/UndoTextReplace.vala ../src/undo_actions/UndoTextTagAdd.vala ../src/undo_actions/UndoTextTagRemove.vala ../src/About.vala ../src/Animator.vala ../src/Application.vala ../src/Bezier.vala ../src/CanvasText.vala ../src/Clipboard.vala ../src/Connection.vala ../src/ConnectionInspector.vala ../src/ConnectionMenu.vala ../src/Connections.vala ../src/ConnectionsMenu.vala ../src/CurrentInspector.vala ../src/Document.vala ../src/DrawArea.vala ../src/DynamicNotebook.vala ../src/EmptyInspector.vala ../src/EmptyMenu.vala ../src/Export.vala ../src/Exporter.vala ../src/Exports.vala ../src/FormattedText.vala ../src/GroupsMenu.vala ../src/HtmlToMarkdown.vala ../src/ImageEditor.vala ../src/ImageManager.vala ../src/Layout.vala ../src/Layouts.vala ../src/LinkDash.vala ../src/LinkType.vala ../src/MainWindow.vala ../src/MapInspector.vala ../src/Node.vala ../src/NodeAlign.vala ../src/NodeBorder.vala ../src/NodeGroup.vala ../src/NodeGroups.vala ../src/NodeImage.vala ../src/NodeInspector.vala ../src/NodeMenu.vala ../src/NodesMenu.vala ../src/NoteView.vala ../src/Partitioner.vala ../src/Preferences.vala ../src/QuickEntry.vala ../src/Selection.vala ../src/Sticker.vala ../src/Stickers.vala ../src/StickerInspector.vala ../src/Style.vala ../src/StyleInspector.vala ../src/Styles.vala ../src/Tagger.vala ../src/TextCompletion.vala ../src/TextMenu.vala ../src/TextParser.vala ../src/Theme.vala ../src/ThemeEditor.vala ../src/Themes.vala ../src/UndoBuffer.vala ../src/UndoItem.vala ../src/UndoTextBuffer.vala ../src/UndoTextItem.vala ../src/UrlEditor.vala ../src/Utils.vala ../tests/assert.vala ../tests/test.vala
../src/NodeMenu.vala:398.21-398.38: error: The type name `Granite.AccelLabel' could not be found
    var task_acc = (Granite.AccelLabel)_task.get_child();
                    ^^^^^^^^^^^^^^^^^^
../src/NodeMenu.vala:399.21-399.38: error: The type name `Granite.AccelLabel' could not be found
    var link_acc = (Granite.AccelLabel)_link.get_child();
                    ^^^^^^^^^^^^^^^^^^
../src/NodeMenu.vala:400.21-400.38: error: The type name `Granite.AccelLabel' could not be found
    var fold_acc = (Granite.AccelLabel)_fold.get_child();
                    ^^^^^^^^^^^^^^^^^^
Compilation failed: 3 error(s), 0 warning(s)
ninja: build stopped: subcommand failed.
Unable to build project, please review log

**Comment by phase1geo** [comment]
There is an outstanding issue to track this problem.  I don't have a solution for the problem at hand.  According to the Vala documents, the Granite.AccelLabel is a supported feature (even in the latest library version as far as I can tell).  I am stuck in elementary OS 5.1 Hera for now, waiting for the 6.0 Odin release to be made available for developers.  I don't have enough information at this point to be of any help as of yet.
